_docs_integrate/03-connector-modules.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Connector Modules"
+permalink: /integrate/connector-modules
+toc: true
+---
+
+Since the Connector is based on the Runtime, [all Modules of the Runtime]({% link _docs_explore/60-runtime.md %}#runtime-modules) are also available in the Connector.
+
+Additionally, the Connector defines its own Modules that only make sense in the context of a Connector and are therefore not defined in the Runtime.
+
+Read more about the Module configuration on the <i class="fas fa-fw fa-cog"/> icon in each title.
+
+### AMQP Publisher <a href="{% link _docs_integrate/11-connector-configuration.md %}#amqppublisher"><i class="fas fa-fw fa-cog"/></a> {#amqppublisher}
+
+This Module proxies all events in the internal event bus of the Connector to an exchange in a configurable AMQP server.
+
+Compared to [webhooks](#webhooksv2)., this gives you the full feature set of a message broker. There are multiple scenarios where this Module outweighs the Webhooks Module. For example:
+
+-   You need persistence for the triggered events.
+-   You want to integrate Enmeshed into an already existing message broker.
+
+### Auto Accept Relationship Creation Changes <a href="{% link _docs_integrate/11-connector-configuration.md %}#autoacceptrelationshipcreationchange"><i class="fas fa-fw fa-cog"/></a> {#autoacceptrelationshipcreationchange}
+
+It is not recommended to use this Module for production szenarios.
+{: .notice--danger}
+
+The `autoAcceptRelationshipCreationChanges` Module listens to the notifications about incoming Relationship Change Requests. It immediately accepts the Requests, using the configured `responseContent`.
```

Comment:
> listens to the notifications

"listens to the EVENTS" (ich würde immer bei der selben Terminologie bleiben, dann kann man als Leser einfacher folgen; sonst fragt man sich hier, was denn jetzt eine "notification" ist)